opt/api/actions/ActionHandler.java

```diff
@@ -0,0 +1,32 @@
+package opt.api.actions;
+
+import java.util.Map;
+
+import opt.api.utils.http.HttpException;
+
+
+public abstract class ActionHandler {
```

Comment:
I'm pretty happy with this + `Interpret` + `Router` - makes it super easy to add new routes and `GET`/`POST` actions super easily by just adding a method or new enum option

Reply:
Yeah, my first attempt was simply a bare-bones server (no POSTs!)